packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts

```diff
@@ -12,6 +12,9 @@
 // https://www.gnu.org/software/classpath/license.html.
 //
 // SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+//
+// Contributors:
+//      Pavel Nikolaev <pnik@1c.ru> (1C-Soft LLC) - Issue #12203
```

Comment:
We don't perform personal committer attributions in this project. Please don't modify the header unless you created the file yourself.

Reply:
Okey, thank you for the advice)